packages/homepage/content/docs/7-environment.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Environment'
+authors: ['Saeris']
+description: 'CodeSandbox uses different execution environments based on what type of Template you use.'
```

Comment:
This is an interesting one. We currently auto-detect the execution environment from the files and folders of a sandbox (eg. dependencies include `vue-cli` -> `vue` environment). So I think it makes more sense to say that we detect it based on the files in a sandbox, maybe something like this?

```suggestion
description: 'CodeSandbox uses different execution environments based on the contents of a sandbox.'
```